src/Kuzzle.ts

```diff
@@ -528,7 +528,7 @@ export class Kuzzle extends KuzzleEventEmitter {
    * @param req
    * @param opts - Optional arguments
    */
-  query (req: RequestPayload = {}, opts: JSONObject = {}): Promise<ResponsePayload> {
+  async query (req: RequestPayload = {}, opts: JSONObject = {}): Promise<ResponsePayload> {
```

Comment:
You cannot use `async` in the SDK because then you need to transpile it for the browser and it will increase it size quite a lot

Reply:
But there not any async anymore...! See my lasts [commit](https://github.com/kuzzleio/sdk-javascript/pull/614/commits/4ed9f7338f351422b896aa2b8564ecc36310d18e#diff-fea3a80abb8ad2b947af003f0230a05d08929cbbeaa5e73d8a1f84e857b4755aR531)